dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthenticationRestControllerIT.java

```diff
@@ -757,4 +775,105 @@ public void testPasswordAuthenticationDoesNotWorkWithShibOnly() throws Exception
             .andExpect(status().isUnauthorized());
 
     }
+
+    @Test
+    public void testShortLivedToken() throws Exception {
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(post("/api/authn/shortlivedtokens"))
+            .andExpect(jsonPath("$.token", notNullValue()))
+            .andExpect(jsonPath("$.type", is("shortlivedtoken")))
+            .andExpect(jsonPath("$._links.self.href", Matchers.containsString("/api/authn/shortlivedtokens")));
+    }
+
+    @Test
+    public void testShortLivedTokenNotAuthenticated() throws Exception {
+        getClient().perform(post("/api/authn/shortlivedtokens"))
+            .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void testShortLivedTokenToDowloadBitstream() throws Exception {
+        Bitstream bitstream = createPrivateBitstream();
+        String shortLivedToken = getShortLivedToken(eperson);
+
+        getClient().perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content?token=" + shortLivedToken))
+            .andExpect(status().isOk());
+    }
+
```

Comment:
can you add a test to verify that a valid short lived token provided for the user A that is not allowed to download the bitstream will be not sufficient for the download? i.e. I like to have a test to proof that is not sufficient to have a generic valid short lived token to download a restricted content

Reply:
this feedback seems to be not yet processed, can you double check please?

Reply:
@abollini The following test verifies the short lived token validity:
* https://github.com/atmire/DSpace/commit/27e733c1e8c557284bb756f32bc09c6a937bc4ee#diff-4906e304a90a1dcbf301b3896d39a353R823

It goes through the following steps:
* A private bitstream is created
* It creates a user that does not have any particular permissions 
* The token for that user is retrieved and passed through to the bitstreams retrieval call
* The verification that the call is forbidden is done

Reply:
Verified this feedback looks to be resolved